Userland/Applications/PixelPaint/ImageEditor.cpp

```diff
@@ -303,7 +303,10 @@ void ImageEditor::second_paint_event(GUI::PaintEvent& event)
 GUI::MouseEvent ImageEditor::event_with_pan_and_scale_applied(GUI::MouseEvent const& event) const
 {
     auto image_position = frame_to_content_position(event.position());
-    auto tool_adjusted_image_position = m_active_tool->point_position_to_preferred_cell(image_position);
+    Gfx::IntPoint tool_adjusted_image_position;
+    if (m_active_tool) {
+        tool_adjusted_image_position = m_active_tool->point_position_to_preferred_cell(image_position);
+    }
```

Comment:
We are returning uninitialized `IntPoint` data when there is no active tool? That doesn't seem right. Maybe you can prevent these events from happening at all as long as there is no active tool?